release_notes.md

```diff
@@ -0,0 +1,67 @@
+ # Release notes
+
+- Woof Mobile APP
+- Release notes version: 1
+- Release note date: May. 4, 2023
+
+All notable changes to this application will be documented in this file.
+
+## Overview
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+## [Unreleased]
+
+`More information about working with the application will be added during development.`
+
+### User impacts
+
+Software requirements: 
+
+ * Deployment target for iOS SDKs - 15.0
+ * Supported devices - iPhone only
+ * Supported modes - portrait
+ * Dark/night theme - deactivate
+ * The linters (Swiftformat and Swiftlint) integrated with Xcode in the build phase.
+
+### Notes
+
+Anyone can download and use the app features.
+
+### Support
+
+You can contact the authors by the following GitHub's usernames:
+
+- Nadia Shpakouskaya: [@NadzeyaShpakouskaya](https://github.com/NadzeyaShpakouskaya)
+- Alex Zavadskiy: [@iBYarchitect](https://github.com/iBYarchitect)
+- Julya Kmita: [@julyakmita](https://github.com/julyakmita)
+
+### Added 
+- Users can now download the initial Xcode project.
```

Comment:
Imagine my father uses this app. He's a user in fact. Can he download the project?


Please discuss in the #general channel, what is the purpose of the release notes and what we should write in the file.

Reply:
Thanks for the comment!
This information has been already discussed in the thread: 
https://ios-engineering-2023.slack.com/archives/C04EDP52CLD/p1676985841295699

For me,  the clearest explanation is the following: 
```
**The release notes detail the changes, upgrades, additional features and bug fixes** associated with the launch. **Release notes can offer valuable information about specific changes** and enhancements, helping customers understand the value of the new product or update. When written successfully, release notes can **help communicate important information to customers** and provide internal value to development teams.
```